### ISSUE TITLE
softmax regression with pytorch

### ISSUE THREAD

**Issue by jusunglee-ai** [jusunglee-ai/jusunglee-ai.github.io#20]
![Untitled 1](https://github.com/jusunglee-ai/jusunglee-ai.github.io/assets/125032849/55cd7138-98a4-4f36-923a-d08a70911064)
![Untitled 2](https://github.com/jusunglee-ai/jusunglee-ai.github.io/assets/125032849/74c88fe0-5171-4d6c-9724-7857a61ad7cb)

![Untitled](https://github.com/jusunglee-ai/jusunglee-ai.github.io/assets/125032849/abfb9a35-601a-43e8-b31a-1d1dac722f4b)
